pulp_docker/app/views.py

```diff
@@ -14,3 +57,235 @@ def get(self, request):
         Return a response to the "GET" action.
         """
         return response.Response({})
+
+
+class ServeContentMixin(object):
+    """
+    A mixin that helps serve content using the enabled web server
+    """
+
+    def _django(self, path, headers):
+        """
+        The content web server is Django.
+
+        Stream the bits.
+
+        Args:
+            path (str): The fully qualified path to the file to be served.
+
+        Returns:
+            StreamingHttpResponse: Stream the requested content.
+
+        """
+        try:
+            file = FileWrapper(open(path, 'rb'))
```

Comment:
I'm not very familiar with the StreamingHttpResponse, but it doesn't look like it closes the file. Could you move line 86 into the try except? That would allow you to use `with open`

Reply:
It's not needed. Django takes care of it.

https://github.com/django/django/blob/master/django/http/response.py#L381
https://github.com/django/django/blob/master/django/http/response.py#L243